runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -317,30 +376,40 @@ public CorfuRuntime registerBeforeRpcHandler(Runnable handler) {
     @Setter
     public Function<String, IClientRouter> getRouterFunction = overrideGetRouterFunction != null
             ? (address) -> overrideGetRouterFunction.apply(this, address) : (address) -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove useless curly braces around statement and then remove useless return keyword [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1602)